src/unlimited.cpp

```diff
@@ -415,9 +434,17 @@ std::string FormatCoinbaseMessage(const std::vector<std::string>& comments,const
 CNode* FindLikelyNode(const std::string& addrName)
```

Comment:
Suggest to rename to more descriptive, e.g. FindNodeByAddr

What is the 'Likely' in name intended to refer to? It's either a match or it isn't, right?

This function would benefit from a unit test and a function header that explains a bit more about the intended purpose and limitations (e.g. it returns on first match in case of wildcard)